### PR TITLE
build: update dependencies

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: packages/primitives/package.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
     "blob-polyfill": "7.0.20220408",
     "esbuild-plugin-alias": "latest",
     "event-target-shim": "6.0.2",
-    "formdata-node": "4.4.1",
+    "formdata-node": "5.0.0",
```
Comment (Kikobeats) [inline packages/primitives/package.json:33]: v5 is dropping node12 and use ESM; we have to be sure ESM is converted to CommonJS by our TS build.

File: packages/primitives/package.json [hunk 2]
```diff
@@ -30,13 +30,13 @@
...
     "text-encoding": "0.7.0",
     "tsup": "6",
-    "undici": "5.11.0",
-    "urlpattern-polyfill": "6.0.1",
+    "undici": "5.13.0",
+    "urlpattern-polyfill": "6.0.2",
     "web-streams-polyfill": "4.0.0-beta.3",
-    "whatwg-url": "11.0.0"
+    "whatwg-url": "12.0.0"
```
Comment (Kikobeats) [inline packages/primitives/package.json:39]: essentially dropping node12

File: packages/primitives/package.json (hunk 1 above)
Comment (Kikobeats) [reply packages/primitives/package.json:33]: tested via REPL:

```
~/vercel/edge-runtime/packages/runtime primitives ≡
❯ node dist/cli/repl.js
ƒ => const formData = new FormData();
undefined
ƒ => formData
FormData { [Symbol(state)]: [] }
ƒ => formData.append('foo', 'bar')
undefined
ƒ => formData
FormData { [Symbol(state)]: [ { name: 'foo', value: 'bar' } ] }
ƒ =>
```

File: packages/primitives/package.json (hunk 2 above)
Comment (Kikobeats) [reply packages/primitives/package.json:39]: not related with the dependency upgrade, but CLI is not printing URL as expected:

```
ƒ => new URL('https://example.com')
URL {  }
ƒ => const url = new URL('https://example.com')
undefined
ƒ => url
URL {  }
ƒ =>
```

**Review by feugy — APPROVED** [review]
👍🏻

**Pull request merged**